pipeline/00-ingest.R

```diff
@@ -147,11 +150,11 @@ recode_column_type <- function(col, col_name, dict = col_type_dict) {
     pull(var_type)
 
   switch(col_type,
```

Comment:
This is a pre-commit change. Do we want to keep it?

Reply:
Just add a line break after the `switch(` call and it should be fixed. See above